### ISSUE TITLE
Incorrect mention of how the variance is calculated in https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm2d.html

### ISSUE THREAD

**Issue by fredpell1** [pytorch/pytorch#105937]
### 📚 The doc issue

From https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm2d.html: The standard-deviation is calculated via the biased estimator, equivalent to torch.var(input, unbiased=False). However, when track_running_stats is True, unbiased is set to True based on the results obtained.
To reproduce:
```py
x = torch.randn(3,3,16,16)
norm = torch.nn.BatchNorm2d(3, track_running_stats=True)
my_var = torch_norm.running_var.clone()
for i in range(3):
   my_var = (1 - norm.momentum) * my_var + norm.momentum * (x ** i).var((0,2,3), unbiased=True)
   out = norm(x ** i)
   print(f"{my_var=}, {norm.running_var=}")
```
Output:
```sh
my_var=tensor([0.9000, 0.9000, 0.9000]), norm.running_var=tensor([0.9000, 0.9000, 0.9000])
my_var=tensor([0.9125, 0.9077, 0.9147]), norm.running_var=tensor([0.9125, 0.9077, 0.9147])
my_var=tensor([1.0278, 0.9946, 1.0586]), norm.running_var=tensor([1.0278, 0.9946, 1.0586])
``` 

### Suggest a potential alternative/fix

Replace "The standard-deviation is calculated via the biased estimator, equivalent to torch.var(input, unbiased=False)" with
"The standard-deviation is calculated via the biased estimator, equivalent to torch.var(input, unbiased=True)"

**Comment by awgu** [comment]
Perhaps, check the `main` docs:
https://pytorch.org/docs/main/generated/torch.nn.BatchNorm2d.html

<img width="897" alt="Screenshot 2023-07-25 at 12 15 48 PM" src="https://github.com/pytorch/pytorch/assets/31054793/08a24940-3508-437a-a1f6-796b618937f7">

This should have been fixed in https://github.com/pytorch/pytorch/pull/97974.

**Comment by fredpell1** [comment]
thanks for pointing this out, looks like there is a discrepancy between the main and stable docs.

**Comment by awgu** [comment]
Yes. I think the stable docs are from PyTorch 2.0 release and will not reflect new changes since that release. The main docs are the ones according to the code in the repo today.

**Comment by awgu** [comment]
I am closing this since the issue has been resolved in main.
